lago/sdk_utils.py

```diff
@@ -0,0 +1,74 @@
+import functools
+import inspect
+import wrapt
+
+
+class SDKWrapper(wrapt.ObjectProxy):
+    """A proxy object that exposes only methods which were decorated with
+    :func:`expose` decorator."""
+
+    def __getattr__(self, name):
+        attr = getattr(self.__wrapped__, name)
+        return filter_sdk(attr, name)
```

Comment:
Went with ``getattr_sdk`` - the reasoning is that it behaves similarly to ``getattr`` - raises ``AttributeError`` or returns the attribute.